package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@thoughtspot/visual-embed-sdk",
-    "version": "1.33.3",
+    "version": "1.36.0",
```

Comment:
Don't change it to 1.36.0. We change it on every new release. If you want to test it out, you can create a version 1.33.4 and use that for your testing purpose.